examples/llm/benchmarks/README.md

```diff
@@ -0,0 +1,143 @@
+<!--
+SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+SPDX-License-Identifier: Apache-2.0
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# LLM Deployment Benchmarking Guide
+
+This guide provides detailed steps on benchmarking Large Language Models (LLMs) in single and multi nodes configurations.
+
+> [!NOTE]
+> We recommend trying out the [LLM Deployment Examples](./README.md) before benchmarking.
+
+## Prerequisites
+
+H100 80GB x8 node(s) are required for benchmarking.
+
+1\. Build benchmarking image
+```bash
+./container/build.sh
+```
+
+2\. Download model
+```bash
+huggingface-cli download neuralmagic/DeepSeek-R1-Distill-Llama-70B-FP8-dynamic
+```
+
+3\. Start NATS and ETCD
+```bash
+docker compose -f deploy/docker_compose.yml up -d
+```
+
+## Disaggregated Single Node Benchmarking
```

Comment:
TODO - have a summary description of the setup - along the lines ...

```
In the following steps we compare X to Y. These were chosen to optimize for Z - for more details on your use case please see the tuning guide () 
```

if the tuning guide for example also exists

Reply:
Sure, added the description.